http/middleware/get_ip_address.go

```diff
@@ -10,8 +10,6 @@ import (
 	"github.com/xy-planning-network/trails"
 )
 
-const IpAddrCtxKey = "trails/middleware/ip-address" // TODO(dlk): change to key provided by app?
-
```

Comment:
🧹 An errant reference in `log_request_test.go` is removed in this PR, so clear out this too.